cmd/synchronization/app/options.go

```diff
@@ -43,7 +45,17 @@ func (p *Providers) UnmarshalJSON(data []byte) error {
 	}
 	depURL, err := url.Parse(rawProviders.Deployinator)
 	p.DeployinatorURL = depURL
-	return errors.Wrap(err, "unable to parse Deployinator URL")
+	if err != nil {
```

Comment:
Error check should be before the `depURL` is used. OR do this instead:
```suggestion
        p.DeployinatorURL, err = url.Parse(rawProviders.Deployinator)
	if err != nil {
```
And same above for sc url (and below)